### DISCUSSION TITLE
[Question] nemo container on ngc

### DISCUSSION THREAD

**Discussion by itzsimpl** [NVIDIA-NeMo/NeMo#3689] [Q&A] [answered]
Is there any specific reason that the nemo:1.6.2 container is not yet available on ngc; (https://catalog.ngc.nvidia.com/orgs/nvidia/containers/nemo/tags). The info says the container was modified on Feb. 14, 2022, but the latest tag is still 1.5.1 from Dec. 15, 2021, even though 1.6.2 was released 11 days ago.

**Comment by titu1994** [comment, marked as answer]
There's been a slight issue with the container release due to the finding of various security vulnerabilities in the base Pytorch 21.12 container. 

Btw, you do not need to await building of Nemo containers, we publish the dockerfile so that users may build their own containers at will as long as they have ngc credentials to pull the base container. You can simply switch to the r1.6.2 branch, and build the dockerfile as per instructions in the readme. If you are comfortable with the security implications, then the container is only a few docker commands away. 

Sidenote, The dockerfile is provided primarily so that users may perform independent research in their forks of Nemo, build a container and train custom code on their own slurm clusters as needed without any delay caused by the Nemo team (precisely for scenarios such as this).
